huxley/api/serializers/school.py

```diff
@@ -74,20 +74,48 @@ class Meta:
 
 
     def validate(self, data):
+        invalid_fields = {}
         international = data.get('international')
         primary_phone = data.get('primary_phone')
         secondary_phone = data.get('secondary_phone')
+        beginner_delegates = data.get('beginner_delegates')
+        intermediate_delegates = data.get('intermediate_delegates')
+        advanced_delegates = data.get('advanced_delegates')
+        spanish_speaking_delegates = data.get('spanish_speaking_delegates')
+        chinese_speaking_delegates = data.get('chinese_speaking_delegates')
 
         if primary_phone:
-            if international:
-                validators.phone_international(primary_phone)
-            else:
-                validators.phone_domestic(primary_phone)
+            try:
```

Comment:
Seems like this can be extracted into a function, since we do the same thing twice.